environment/docker/apex/dockerfile

```diff
@@ -10,7 +10,7 @@ RUN git clone https://github.com/NVIDIA/apex && \
     cd apex && \
     pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
 
-RUN pip install toolz pytorch-ignite torchvision pandas opencv-python fire tensorboardx scikit-learn yacs
+RUN pip install toolz pytorch-ignite torchvision pandas fire tensorboardx scikit-learn yacs
```

Comment:
Do we need to modify apex as a part of this PR?

Reply:
are we using apex? I'm not sure, but i figured opencv would be removed from all the repo, so there would be no need to install it

Reply:
ok - doesn't do anything - please feel free to leave